sdk/python/kfp/_client.py

```diff
@@ -1410,6 +1411,8 @@ def list_pipeline_versions(
           page_token: Token for starting of the page.
           page_size: Size of the page.
           sort_by: One of 'field_name', 'field_name desc'. For example, 'name desc'.
+          filter: A url-encoded, JSON-serialized Filter protocol buffer
```

Comment:
Is it possible to give an example of what a filter value could be?

Reply:
I've added an example to all the affected areas. Looks like the URL encoding actually happens under the hood as a part of the REST client in `kfp-server-api` so technically you don't need to provide a URL encoded filter, I can update the docstring if that works?

Also in the comment I'm referring to a private dictionary interface which holds the mapping between `op` name and index, not ideal but happy to make that interface public if you want? We could expose it via `from kfp import FILTER_OPERATIONS` or something like that?

Reply:
I think that's good enough for now. Thanks!